capa/ida/plugin/form.py

```diff
@@ -275,6 +282,9 @@ def __init__(self, name):
 
         self.Show()
 
+        if (option & Options.ANALYZE) == Options.ANALYZE:
```

Comment:
the argument here is being used like a bitflag, but everywhere else is described as a number. i guess thats not technically wrong, but we'd need to update more places as soon as we add a second bit.

Reply:
we could also use it as a number here until we add another option. Maybe we never add a second option and this is a bit confusing with one option. That's what I meant [before](https://github.com/fireeye/capa/pull/548#discussion_r625910367).

Reply:
yes, and suggested changes above update the documentation to describe this as a bitflag.